### PR TITLE
React intl cleanup groupby msgs

### PR DESCRIPTION
**Pull request by ddonahue007**
Issue: https://issues.redhat.com/browse/COST-1372

Refactor on the groupby messages in the json.messages file. This is not changing the test we see but how we use the messages themselves.

### CODE REVIEW

File: locales/data.json [hunk 1]
```diff
@@ -81,16 +81,8 @@
...
     "GroupByTop": "DE Top {groupBy}s",
-    "GroupByTopValueAccount": "DE accounts",
-    "GroupByTopValueCluster": "DE clusters",
-    "GroupByTopValueInstanceType": "DE Instance types",
-    "GroupByTopValueNode": "DE Nodes",
-    "GroupByTopValueOrganizationalUnitID": "Organizational units",
-    "GroupByTopValuePod": "DE Pods",
-    "GroupByTopValueProject": "DE Projects",
-    "GroupByTopValueRegion": "DE Regions",
-    "GroupByTopValueService": "DE Services",
```
Comment (dlabrecq) [inline locales/data.json:92]: Were these keys unused? 

Can we please make GroupByTop look similar to GroupByValues? The group_by key is not localized.

Screen Shot 2021-08-19 at 10 27 43 AM

**Review by dlabrecq — COMMENTED** [review]
Can we please make GroupByTop look similar to GroupByValues? The group_by key is not localized.

You can see this issue best with the Azure overview.

Screen Shot 2021-08-19 at 10 27 43 AM

**Review by dlabrecq — CHANGES_REQUESTED** [review]
This appears to be broken

Screen Shot 2021-08-19 at 10 23 20 AM

File: src/locales/messages.ts [hunk 2]
```diff
@@ -495,57 +495,32 @@ export default defineMessages({
...
-  GroupByTopValueRegion: {
-    defaultMessage: 'EN Regions',
-    description: 'Regions',
-    id: 'GroupByTopValueRegion',
-  },
-  GroupByTopValueService: {
-    defaultMessage: 'EN Services',
-    description: 'Services',
-    id: 'GroupByTopValueService',
-  },
```
Comment (dlabrecq) [inline src/locales/messages.ts:542]: Were these keys unused?

Comment (ddonahue007) [reply src/locales/messages.ts:542]: Yes

File: src/utils/dateRange.ts [hunk 3]
```diff
@@ -56,7 +56,7 @@ export function getForDateRangeString(
 
   return intlHelper(
     intl.formatMessage(message, {
-      groupByValue: intl.formatMessage(messages.GroupByValues, { value, count: 2 }),
+      groupByValue: intl.formatMessage(messages.GroupByValuesTitleCase, { value, count: 2 }),
```
Comment (dlabrecq) [inline src/utils/dateRange.ts:59]: This isn't always a group by value. For the details pages, this is a cost.
I believe this is why the key was previously named `value`

**Review by dlabrecq — APPROVED** [review]

**Pull request merged**